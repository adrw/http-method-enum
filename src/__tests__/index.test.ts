import deepSortObject from 'deep-sort-object'
import { HTTPMethod } from '../index'

test('HTTPMethod', () => {
  expect(HTTPMethod).toMatchSnapshot()
  expect(JSON.stringify(deepSortObject(HTTPMethod), undefined, 2)).toEqual(
    JSON.stringify(HTTPMethod, undefined, 2)
  )
})
