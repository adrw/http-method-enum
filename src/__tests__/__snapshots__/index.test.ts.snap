// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HTTPMethod 1`] = `
Object {
  "CONNECT": "CONNECT",
  "DELETE": "DELETE",
  "GET": "GET",
  "HEAD": "HEAD",
  "OPTIONS": "OPTIONS",
  "PATCH": "PATCH",
  "POST": "POST",
  "PUT": "PUT",
  "TRACE": "TRACE",
}
`;
